css/style.css

```diff
@@ -0,0 +1,168 @@
+@import url("https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap");
+* {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  font-family: "Ubuntu", sans-serif;
+}
+body {
+  background: #f8f9fa;
+}
+h1 {
+  text-align: center;
+  font-size: 40px;
+  font-weight: 500;
+  color: #3b3663;
```

Comment:
do  not  apply   color directly    to the tag instead  create a  class  and then gave element class